doc/ReleaseNotes/previous/v6.4.2.tex

```diff
@@ -10,7 +10,7 @@
+		\item The Input Data Processor (IDP) is introduced in this release to read user-provided input files and store user-provided input data in memory for subsequent use by simulation, model, and package components.  Components that have been integrated with IDP no longer handle input files directly but rather retrieve all input data from named locations, called memory paths, allocated in managed memory.  The collection of all simulation input data in managed memory is called the input context.  IDP uses existing descriptions of input variables, called variable definitions, to interpret and store input.  The program variable definition set and its representation in the input context is described as the Input Data Model (IDM).  Input variables can be recognized in a memory dump (e.g., with the MEMORY\_PRINT\_OPTION activated in the options block of mfsim.nam) by their memory path prefix string "\_\_INPUT\_\_".  Components that later access input typically copy data from the input context to their own memory space; therefore, the present implementation of the IDP results in an increased memory footprint for the program.  Among its advantages include the consolidation of all input processing early in program runtime, and outside of any particular component, enabling the support of alternative types of input data sources.  Input file types that are currently processed by IDP include DIS6, DISU6, DIV6, NPF6, DSP6, and Name File inputs for the Simulation (mfsim.nam) and GWF and GWT models.
```

Comment:
For some reason, the GH UI does not highlight the one change here "implemention" -> "implementation"